src/Extensions/Script/Serialization/JavaScriptConverter.cs

```diff
@@ -0,0 +1,17 @@
+// MIT License.
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Web.Script.Serialization;
+public abstract class JavaScriptConverter
+{
+    public abstract IEnumerable<Type> SupportedTypes
+    {
+        get;
+    }
+
+    public abstract object Deserialize(IDictionary<string, object> dictionary, Type type, JavaScriptSerializer serializer);
+
+    [SuppressMessage("Microsoft.Naming", "CA1720:IdentifiersShouldNotContainTypeNames", MessageId = "obj",
+        Justification = "Cannot change parameter name as would break binary compatibility with legacy apps.")]
```

Comment:
Can you create a constant kind of like https://github.com/dotnet/systemweb-adapters/blob/main/src/Microsoft.AspNetCore.SystemWebAdapters/Constants.cs#L10 for this kind of suppression?

Reply:
Doesn't look like you're using the constants